src/contextgpt_structs.rs

```diff
@@ -15,7 +15,7 @@ impl FromStr for RequestTypeOptions {
             "author" => Ok(RequestTypeOptions::Author),
             "file" => Ok(RequestTypeOptions::File),
             _ => Err(format!(
-                "Could not parse the request type: {}",
+                "Could not parse the request type: {}. Available options are: author, file",
```

Comment:
While this works right now, the problem is - that if we end up adding one more option, we'll always have to update this message.

Could you think of something where we could have it dynamically generated from the `RequestTypeOptions` enum?

Reply:
can you help me with this, I have created a vector and I have added all the RequestTypeOptions to that vector and I am displaying it from there only,

due to which LSP is showing this warning
![image](https://github.com/krshrimali/context-pilot-rs/assets/79051850/714dcf8d-8e76-4d2b-bede-220e37d886ad)

Reply:
I am unable to come up with a solution in which we could dynamically show our enum, can you guide me a bit, the solution I have committed is that we have to manually add all the enums into our vector and then print it out

Reply:
Lemme try doing this live and share the solution with you, happy for you to just pick it up from there and try committing it :)

Thanks for making efforts with this, please don't close the PR - let's just modify a few things later and get this in. I appreciate you taking time to contribute!